sql/parser/parse_test.go

```diff
@@ -614,6 +614,13 @@ CREATE TABLE a (b INT DEFAULT (SELECT 1))
                                         ^
 `,
 		},
+		{
+			`SELECT POSITION('high', 'a')`,
```

Comment:
Yeah, so that is really a kind of bug in that you're parsing this into a simple function call. We probably should have a new Expr that can `String()` out to this correctly.